3-projects/main.tf

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_folder" "app" {
```

Comment:
Does this belong in this component?

Reply:
I believe this is part of the basic example and where the `prod` and `nonprod` folders will be created beneath it

Reply:
On closer inspection, this should be an example business unit or team @ericyz , it should also probably live in the same file as the example project/s